apt.conf.d/Makefile

```diff
@@ -0,0 +1,13 @@
+sysconfdir ?= /etc/apt.conf.d/
```

Comment:
Should this path be `/etc/apt/apt.conf.d` ?

Reply:
Good catch.